cmd/flags_core.go

```diff
@@ -73,7 +73,7 @@ func validateAddress(address string) (string, string, error) {
 		return "", "", fmt.Errorf("both protocol and host must present in the address")
 	}
 
-	if _, port, err := net.SplitHostPort(u.Host); err != nil || port == "" {
+	if _, _, err := net.SplitHostPort(u.Host); err != nil {
```

Comment:
This check to ensure port is not empty is unnecessary, it will fail if port is empty.